Makefile

```diff
@@ -13,7 +13,7 @@ test-verbose:
 add-target:
 	rustup target add wasm32-unknown-unknown
 
-build:
+build: add-target
```

Comment:
Do we need to do this at every build?

If using online IDE, we have already done this in Gitpod config. I suppose similarly can be done with Github Codespace. If doing local development, a NEAR developer should be aware of this target.

Reply:
This operation is idempotent. If it was added before, running it again won't cause any problem. In contrast, developers new to the ecosystem won't usually know that we need to do this before we're able to `cargo build` successfully. I got bumped twice actually so far, and it took me a while to realise I need to do this :) Doing this also helps with code readability and educational purposes!